docs/setup-guide.md

```diff
@@ -29,6 +29,14 @@ requirements:
   recommend a default range of 49152-49251 on each host, though you can
   adjust this to fit your needs.
 
+  * The following kernel modules must be loaded:
+  
+      1. dm_snapshot
+      2. dm_mirror
+      3. dm_thin_pool
+     
+   For kernel modules, `lsmod | grep <name>` will show you if a given module is present, and `modprobe <name>` will load a given module.
```

Comment:
This line is not indented properly. Try indenting it by two spaces, otherwise make it a sub-item right after `3.`.